website/styles.css

```
@@ -0,0 +1,131 @@
+.title {
+    min-height: 200px;
+}
+
+.title-photo {
+    background: url("assets/cribbity_crob.jpg");
+    width: 100%;
+    height: 100%;
+    /* this will look awful, but whatever... */
+    background-size: contain;
+}
+
+.center-banner {
+    padding: 50px;
+    background: rgba(0, 0, 0, 0.75);
+}
+
+.center-banner-text {
+    background: none;
+    opacity: 1.0;
+    font-family: "Faster One", serif;
+    color: chartreuse;
+    font-size: 80px;
+    text-align: center;
+}
+
+.menu {
+    background: lime;
+}
+
+.menu-item {
+    background: magenta;
+    border-radius: 20px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    font-family: 'Ranchers', sans-serif;
+    font-size: 36px;
+    text-align: center;
+}
+
+.menu-item:hover {
+    background: cyan;
+    cursor: pointer;
+}
+
+.content {
+    min-height: 300px;
+}
+
+#getCrows {
+    background: lavenderblush;
+}
+
+#getFacts {
+    background: lemonchiffon;
+}
+
+#teachUs {
+    background: lightcyan;
```

```suggestion
    background: azure;
```
`0xE0FFFF` vs `0xF0FFFF`, similar enough

---

lol, aight

---

hehe